### ISSUE TITLE
Is this Project still alive and maintained?

### ISSUE THREAD

**Issue by makupi** [pyserial/pyserial-asyncio#55]
It seems like there wasn't any activity in this repository in the last couple years. Is there anyone else maintaining a fork or some other async implementation of serial?

**Comment by rob-smallshire** [comment]
What is it you want that doesn't work?

**Comment by makupi** [comment]
I'm starting a project that requires an async serial connection, this seems to be the best route as aioserial basically just runs read/write within executors. But I'm worried that upcoming Issues won't get fixed at all if no one is maintaining it anymore.

**Comment by rob-smallshire** [comment]
It's intentionally a really very simple thing. There isn't much code, and so there isn't much to fix.

**Comment by septatrix** [comment]
Personally I would assume that once this package is ready it would be either merged into the upstream `pyserial` or at least get a `1.0.0` release. However the current version is `0.3.0` and there does not seem to be too much activity here...

**Comment by rob-smallshire** [comment]
I'm not able to make releases to PyPI. @zsquareplusc can you cut a new release of pyserial-asyncio, please? There have been quite a few important fixes and small improvements since the last release.

**Comment by rob-smallshire** [comment]
@septatrix pyserial-asyncio was originally demerged from pyserial because core pyserial had much broader and more stringent compatibility requirements than the asyncio part.

**Comment by gwww** [comment]
Is is worth renaming this and creating a  PyPi release under a new name?
Would @zsquareplusc consider putting in a new PyPi maintainer?
Are there other options?

**Comment by septatrix** [comment]
> @septatrix pyserial-asyncio was originally demerged from pyserial because core pyserial had much broader and more stringent compatibility requirements than the asyncio part.

However once this is considered stable I think it would be best to included this in the main `pyserial` package

**Comment by septatrix** [comment]
It may also deter people from using this package if it is referred as experimental everywhere. Furthermore there are certainly incidents where this has lead to confusion as can be seen by #22 (it does not help that the documentation at pythonhosted still exists but I will open an issue for that at the main package).

Additionally the documentation is far from complete and only barely useful. Most information about this package for my usage was gathered from a blog post and studying the source code which I am sure not everyone is willing to do.

**Comment by rob-smallshire** [comment]
I agree with all of your points. It's a team effort to improve them. I'm only here because I rewrote the original async serial effort to a) work, and b) not be tied to Python 3.4 yield syntax.

**Comment by rob-smallshire** [comment]
Right now we have a bunch of open issues, but only one PR for which the only (!) test doesn't pass. In my current work, I'm not using pyserial-asyncio so I can only contribute occasional hobby time. I'm more than happy to review and merge contributions.
